src/_derive/mod.rs

```diff
@@ -213,6 +213,8 @@
 //! - `subcommand`: Delegates definition of subcommands to the field (must implement
 //!   [`Subcommand`][crate::Subcommand])
 //!   - When `Option<T>`, the subcommand becomes optional
+//! - `global = <bool>`: [`Arg::global`][crate::Arg::global]
+//!   - When not present: only reads the argument from the specified level of command (no sub-commands).
```

Comment:
This is a "raw" attribute, rather than a "magic" attribute and is documented in the above section